modules/core/server/views/email-templates/email.server.view.html

```diff
@@ -171,7 +171,7 @@
                                                                   <td valign="middle" align="center" style="text-align:center">
                                                                     <center>
                                                                       {% if headerUrl %}<a href="{{headerUrl}}" target="_blank" rel="noopener" class="logo-link">{% endif %}
-                                                                          <h2 class="logo-text">TRUSTROOTS</h2>
+                                                                          <h2 class="logo-text">FEDERATED_PLATFORM_NAME</h2>
```

Comment:
I don't see FEDERATED_PLATFORM_NAME being referenced anywhere else. Should this be a variable of some sort or is it just a random placeholder string here?

Reply:
yep, random place holder - could not manage anything better. "documented" in the wiki, I hope someone does a better job...